src/modelgauge/tests/safe_v1.py

```diff
@@ -213,11 +215,4 @@ class SafeTestVersion1(BaseSafeTestVersion1):
     class PrivateSafeTestVersion1(BaseSafeTestVersion1):
         annotators = PRIVATE_ANNOTATOR_SET
 
-    for hazard in SafeTestVersion1.hazards:
-        TESTS.register(
-            SafeTestVersion1,
-            f"safe-{hazard}-1.0-private",
-            hazard,
-            Locale.EN_US,
-            ALL_PERSONAS,
-        )
+    register_tests(PrivateSafeTestVersion1, "-private")
```

Comment:
I think calling these "private" on their own is confusing, because they're using public prompts but the private annotator. Let's try to work both dimensions in.

Reply:
Changed to -private_eval